client/components/navbar.jsx

```diff
@@ -23,16 +24,25 @@ class Nav extends React.Component {
     });
   }
 
+  handleClick(event) {
+    this.props.history.push('/bookmarks');
+  }
+
   render() {
     return (
       <>
         <Navbar className='navbar-custom d-flex' expand="lg" sticky="top">
-          <div className="container-fluid">
+          <div className="container-fluid align-content-center">
             <Navbar.Brand href="/" className='orange nav-font'>
               <div className="orange font-raleway">
-                <i className="fa-brands fa-btc orange" />itcoin Exlorer
+                <i className="fa-brands fa-btc" />itcoin Exlorer
               </div>
             </Navbar.Brand>
+            <button className='nav-bookmark-btn nav-font' onClick={this.handleClick}>
+              <a href='/bookmarks'>
+                <i className='fa-solid fa-star orange nav-bookmark-btn'></i>
+              </a>
+            </button>
```

Comment:
Usually we use a Link instead of a button for this
https://v5.reactrouter.com/web/api/Link

You can style it to look like a button, but we don't use buttons for navigation.